client/app.jsx

```diff
@@ -57,6 +57,7 @@ export default class App extends React.Component {
         <>
         {route.path !== 'search-page' &&
         route.path !== 'advanced-search' &&
+        route.path !== '' &&
```

Comment:
When _does_ the header get included? At this point it might make more sense to just explicitly import it into those page components and clean up this `&&` stack.